### DISCUSSION TITLE
Condition on a step in a series of actions

### DISCUSSION THREAD

**Discussion by SantoshSrinivas79** [Budibase/budibase#6275] [Help]
Hello, I want to put a check on Step 2 on upload file (so that it executes only if the value is not empty).  May need conditions for other steps too. Any way to achieve this?

<img width="303" alt="Screenshot 2022-06-11 at 8 21 01 AM" src="https://user-images.githubusercontent.com/1036163/173169890-6459761a-1b8f-4f11-8d7a-11ce7653d85f.png">

**Comment by melohagan** [comment]
Hi @SantoshSrinivas79 

Try using the **Continue/Stop if** action. It would be placed between the validation form and upload actions.
